routes/users.js

```diff
@@ -53,24 +66,46 @@ router.post("/signup", async (req, res) => {
       passwordInput: password,
       confirmPasswordInput,
       ageInput: age,
-      userNameInput: userName
+      userNameInput: userName,
     } = req.body;
 
-    validationsForCreateUser(firstName.trim(), lastName.trim(), emailAddress.trim(), password, Number(age), userName);
-
-    if (password !== confirmPasswordInput) 
-      return res.status(400).render('users/signup', { title: 'Signup', error: "400 - Error: Both password & confirm password should match." });
-
-    let newUser = await createUser(firstName, lastName, emailAddress, password, age, userName);
+    validationsForCreateUser(
+      firstName.trim(),
+      lastName.trim(),
+      emailAddress.trim(),
+      password,
+      Number(age),
+      userName
+    );
+
+    if (password !== confirmPasswordInput)
+      return res.status(400).render("users/signup", {
+        title: "Signup",
+        error: "400 - Error: Both password & confirm password should match.",
+      });
+
+    let newUser = await createUser(
+      firstName,
+      lastName,
+      emailAddress,
+      password,
+      age,
+      userName
+    );
 
     if (newUser.insertedUser) return res.redirect("users/login");
   } catch (err) {
     console.error(err);
 
-    if (typeof err === "string") 
-      return err.startsWith("VError") ? 
-        res.status(400).render('users/signup', { title: "Signup", error: `400 - ${err.substr(1)}`}) : 
-        res.status(400).render('users/signup', { title: "Signup", error: `400 - ${err}` });
+    if (typeof err === "string")
+      return err.startsWith("VError")
+        ? res.status(400).render("users/signup", {
+            title: "Signup",
+            error: `400 - ${err.substr(1)}`,
+          })
+        : res
+            .status(400)
+            .render("users/signup", { title: "Signup", error: `400 - ${err}` });

```

Comment:
This formatting is making the code hard to read
Why are we re-formatting these files ?